well-known-bots.json

```diff
@@ -2288,6 +2290,7 @@
   {
     "id": "apple-crawler",
     "categories": [
+      "ai",
```

Comment:
[AppleBot](https://support.apple.com/en-us/119829) is also used for search. I could see someone adding a deny rule for AI and then being confused about why Apple search isn't working, even though they don't distinguish.

Maybe we need a special docs section on AI to warn about certain AI bots also having dual use e.g. in the case of AppleBot. Then we can show [an example of filtering categories](https://docs.arcjet.com/bot-protection/reference/nextjs#filtering-categories) to remove Applebot, but still filter AI.

Reply:
Dark Visitors says that AppleBot is AI because it trains Siri: https://darkvisitors.com/agents/applebot

Reply:
My guess is people are more likely to be annoyed that they don't show up in Siri / Safari search results than it training AI. So until we hear otherwise we shouldn't categorize it as AI.

Reply:
```suggestion
```